src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/ComEventsHelperTests.cs

```diff
@@ -11,10 +11,18 @@ public class ComEventsHelperTests
     {
         [Fact]
         [PlatformSpecific(TestPlatforms.Windows)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Throws PlatformNotSupportedException in UapAot")]
         public void Combine_NullRcwWindows_ThrowsArgumentNullException()
         {
             AssertExtensions.Throws<ArgumentNullException>(null, () => ComEventsHelper.Combine(null, Guid.Empty, 1, null));
         }
+
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.UapAot, "Throws PlatformNotSupportedException in UapAot")]
+        public void Combine_NullRcwUapAot_PlatformNotSupportedException()
```

Comment:
Ideally yes you're right but practically I don't think this is worth the time right now as its a different repo..